kratos/python/add_processes_to_python.cpp

```diff
@@ -157,7 +157,7 @@ void  AddProcessesToPython(pybind11::module& m)
     ;
 
     py::class_<FindNodalNeighboursProcess, FindNodalNeighboursProcess::Pointer, Process>(m,"FindNodalNeighboursProcess")
-            .def(py::init<ModelPart&, unsigned int, unsigned int>())
+            .def(py::init<ModelPart& >())
```

Comment:
since this process is becoming deprecated, I suggest to leave the old constructor an issue a deprecation-warning saying that this process should be replaced by either of the specific ones

Reply:
ok, it makes sense.

I just wonder if it should be a warning or an error

Reply:
I would say warning is enough, since whoever uses it will have to use another process in the future (i.e. at least some refactor is necessary, not just e.g. changing the name)